### DISCUSSION TITLE
How to build a debug version when developing locally？

### DISCUSSION THREAD

**Discussion by 2239559319** [swc-project/swc#7611] [Q&A] [answered]
The [CONTRIBUTING](https://github.com/swc-project/swc/blob/main/CONTRIBUTING.md) only talk about how to test locally. How to build a debug version locally? Is there a corresponding command ？

**Comment by coder-xiaotian** [comment]
I also want to know how to print log in my swc plugin, but it seems no one can give me an answer.😫

**Comment by kdy1** [comment, marked as answer]
You can do `yarn build:dev`
